### PR TITLE
unreal sdk news

### PR DESCRIPTION
**Pull request by kp-cat**
### Description

Unreal SDK news + Unreal docs update

### Notes for QA

Test the links

### Requirement checklist

- [x] I have validated my changes on a test/local environment.
- [x] I have added my changes to the V1 and V2 documentations.
- [ ] I have checked the SNYK/Dependabot reports and applied the suggested changes.
- [ ] (Optional) I have updated outdated packages.

### CODE REVIEW

**Review by laliconfigcat — APPROVED** [review]

**Review by laliconfigcat — DISMISSED** [review]

**Review by laliconfigcat — APPROVED** [review]

File: website/docs/sdk-reference/unreal.mdx [hunk 1]
```diff
@@ -24,19 +24,19 @@ export const CPPSchema = require('@site/src/schema-markup/sdk-reference/cpp.json
...
-Run the following commands in the root folder of your project (where the `.uproject` file is located):
+To download:
 
-```cmd
-mkdir -p Plugins
-cd Plugins
-git clone https://github.com/configcat/unreal-engine-sdk ConfigCat
-```
+- Create a `Plugins` folder in the root folder of your project (where the `.uproject` file is located).
+- Head to the download the latest [ConfigCat.zip](https://github.com/configcat/unreal-engine-sdk/releases/latest/download/ConfigCat.zip) archive. (or any of the previous versions from: [Releases](https://github.com/configcat/unreal-engine-sdk/releases))
```
Comment (laliconfigcat) [inline website/docs/sdk-reference/unreal.mdx:36]: Just a suggestion, I think it is not necessary to put the other part in brackets.
```suggestion
- Head to the download the latest [ConfigCat.zip](https://github.com/configcat/unreal-engine-sdk/releases/latest/download/ConfigCat.zip) archive or any of the previous versions from: [Releases](https://github.com/configcat/unreal-engine-sdk/releases)
```

File: website/docs/sdk-reference/unreal.mdx [hunk 2]
```diff
@@ -79,7 +77,7 @@ You can configure all ConfigCat related settings inside the `ProjectSettings ->
 </TabItem>
 <TabItem value="cpp" label="C++">
 
-Add Depedency to your `.Build.cs` file:
+Add Dependency to your `.Build.cs` file:
```
Comment (laliconfigcat) [inline website/docs/sdk-reference/unreal.mdx:80]: Not sure about this.
```suggestion
Add the ConfigCat Dependency to your `.Build.cs` file:
```

File: website/versioned_docs/version-V1/sdk-reference/unreal.mdx [hunk 3]
```diff
@@ -30,19 +30,19 @@ This documentation applies to the **v1.x version** of the ConfigCat Unreal SDK.
...
-Run the following commands in the root folder of your project (where the `.uproject` file is located):
+To download:
 
-```cmd
-mkdir -p Plugins
-cd Plugins
-git clone https://github.com/configcat/unreal-engine-sdk ConfigCat
-```
+- Create a `Plugins` folder in the root folder of your project (where the `.uproject` file is located).
+- Head to the download the latest [ConfigCat.zip](https://github.com/configcat/unreal-engine-sdk/releases/latest/download/ConfigCat.zip) archive. (or any of the previous versions from: [Releases](https://github.com/configcat/unreal-engine-sdk/releases))
```
Comment (laliconfigcat) [inline website/versioned_docs/version-V1/sdk-reference/unreal.mdx:42]:
```suggestion
- Head to the download the latest [ConfigCat.zip](https://github.com/configcat/unreal-engine-sdk/releases/latest/download/ConfigCat.zip) archive or any of the previous versions from: [Releases](https://github.com/configcat/unreal-engine-sdk/releases)
```

File: website/versioned_docs/version-V1/sdk-reference/unreal.mdx [hunk 4]
```diff
@@ -85,7 +83,7 @@ You can configure all ConfigCat related settings inside the `ProjectSettings ->
 </TabItem>
 <TabItem value="cpp" label="C++">
 
-Add Depedency to your `.Build.cs` file:
+Add Dependency to your `.Build.cs` file:
```
Comment (laliconfigcat) [inline website/versioned_docs/version-V1/sdk-reference/unreal.mdx:86]: not 100% sure
```suggestion
Add the ConfigCat Dependency to your `.Build.cs` file:
```

**Review by laliconfigcat — CHANGES_REQUESTED** [review]

**Review by laliconfigcat — DISMISSED** [review]

**Review by z4kn4fein — APPROVED** [review]

**Pull request merged**